editor/src/clj/editor/render_pb.clj

```diff
@@ -70,43 +70,43 @@
                          :type :resource
                          :filter "render_script"
                          :label "Script"}
-                        {:path [:named-materials]
+                        {:path [:named-render-resources]
                          :type :table
-                         :label "Materials"
+                         :label "Render Resources"
                          :columns [{:path [:name]
                                     :label "Name"
                                     :type :string
-                                    :default "New Material"}
-                                   {:path [:material]
-                                    :label "Material"
+                                    :default "New Render Resource"}
+                                   {:path [:path]
+                                    :label "Render Resource"
                                     :type :resource
-                                    :filter "material"
+                                    :filter ["material" "render_target"]
                                     :default nil}]}]}]})
 
 (defn- set-form-op [{:keys [node-id] :as user-data} path value]
   (condp = path
     [:script]          (g/set-property! node-id :script value)
-    [:named-materials] (let [graph-id (g/node-id->graph-id node-id)]
-                         (g/transact
-                           (concat
-                             (for [[named-material-id _] (g/sources-of node-id :named-materials)]
-                               (g/delete-node named-material-id))
-                             (for [{:keys [name material]} value]
-                               (make-named-material-node graph-id node-id name material)))))))
-
-(g/defnk produce-form-data [_node-id script-resource named-materials]
+    [:named-render-resources] (let [graph-id (g/node-id->graph-id node-id)]
+                              (g/transact
+                                (concat
+                                  (for [[named-render-resource-id _] (g/sources-of node-id :named-render-resources)]
+                                    (g/delete-node named-render-resource-id))
+                                  (for [{:keys [name path]} value]
+                                    (make-named-render-resource-node graph-id node-id name path)))))))
```

Comment:
Indentation looks messed up here. Select lines, then **Code > Auto-Indent Lines** should fix it!